### PR TITLE
Datastore branch

### PR DESCRIPTION
**Pull request by tonioshikanlu**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -19,14 +19,61 @@
...
+    List<String> tasks = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      String comment = (String) entity.getProperty("comment");
+
+      tasks.add(comment);
+    }
+  /** Convert to json string using gson*/
+    Gson gson = new Gson();
+  	String json = gson.toJson(tasks);
+  	response.setContentType("application/json;");
```
Comment (charu0291) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:59]: code indentation?

Comment (tonioshikanlu) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:59]: thank you will note that.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,61 @@
...
+    response.getWriter().println(json);
+
+  }
+ @Override
+ public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Accepts comment and updates list to show messages.
+    String commentString = request.getParameter("comment");
```
Comment (curves4d) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:66]: Is this coming as a query param in a POST request?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (charu0291) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:59]: identation

**Pull request merged**